src/core/mod.rs

```diff
@@ -7,6 +7,7 @@ pub mod indexer;
 pub mod matter;
 pub mod sadder;
 pub mod saider;
+pub mod seqner;
```

Comment:
You should also check out `src/lib.rs` and add a pub use statement for `Seqner`.

Reply:
Done. Thanks for catch!